config/test.exs

```diff
@@ -1,7 +1,11 @@
 import Config
 
 config :dotcom, :cache, Dotcom.Cache.TestCache
-config :dotcom, :redis, Redis.Mock
-config :dotcom, :redix, Redix.Mock
-config :dotcom, :redix_pub_sub, Redix.PubSub.Mock
+config :dotcom, :redis, Dotcom.Redis.Mock
+config :dotcom, :redix, Dotcom.Redix.Mock
+config :dotcom, :redix_pub_sub, Dotcom.Redix.PubSub.Mock
```

Comment:
Renaming these so they match the pattern for other config using `:dotcom` as the app.